src/layers/Notification/Notifications.tsx

```diff
@@ -94,19 +100,33 @@ export const Notifications: FC<NotificationsProps> = ({
 
   const notifyError = useCallback(
     (title: string, options: NotificationOptions = {}) =>
-      notify(title, { ...options, variant: 'error' }),
+      notify(title, {
+        variant: 'error',
+        icon: <ErrorCircleIcon />,
+        ...options
+      }),
     [notify]
   );
 
   const notifyWarning = useCallback(
     (title: string, options: NotificationOptions = {}) =>
-      notify(title, { ...options, variant: 'warning' }),
+      notify(title, { variant: 'warning', icon: <WarningIcon />, ...options }),
     [notify]
   );
 
   const notifySuccess = useCallback(
     (title: string, options: NotificationOptions = {}) =>
-      notify(title, { ...options, variant: 'success' }),
+      notify(title, {
+        variant: 'success',
+        icon: <CheckCircleIcon />,
```

Comment:
These need to be where you can pass them in

Reply:
you can pass in whatever icon you want using the options in `notify`, `notifySuccess`, etc

```
notify("Notification Title", { icon: <CustomIcon />})
```

it'll overwrite this default icon through the spread `...options` below. Is that what you mean?

Reply:
Pass the icons as optional props too?

```
<Notifications
  icons: {
    info: <InfoIcon />
    error: <ErrorIcon />
    warning: <WarningIcon />
    success: <SuccessIcon />
  }
>
```